SpatialGDK/Source/SpatialGDK/Public/Interop/RPCs/CrossServerRPCService.h

```diff
@@ -74,7 +77,8 @@ class SPATIALGDK_API CrossServerRPCService
 
 	ActorCanExtractRPCDelegate CanExtractRPCDelegate;
 	ExtractRPCDelegate ExtractRPCCallback;
-	const FSubView* SubView;
+	const FSubView* ActorSubView;
+	const FSubView* WorkerEntitySubView;
```

Comment:
Why can't these be references, given that they're passed as such on construction and can never become null after?